### PR TITLE
add lineplot, relplot, scatterplot

### PR DESCRIPTION
**Pull request by josephmarturano**
My attempt to add lineplot, relplot, and scatterplot as discussed in issue #12 . This is my first pull request so there is a good chance I did something incorrectly.

### CODE REVIEW

File: src/Seaborn.jl [hunk 1]
```diff
@@ -60,7 +63,7 @@ macro delegate(f_list...)
     blocks
 end
 
-@delegate jointplot pairplot kdeplot lmplot regplot residplot interactplot coefplot boxplot violinplot stripplot swarmplot pointplot clustermap tsplot palplot FacetGrid PairGrid JointGrid axes_style set_style plotting_context set_context set_color_codes reset_defaults reset_orig set
+@delegate lineplot scatterplot relplot jointplot pairplot kdeplot lmplot regplot residplot interactplot coefplot boxplot violinplot stripplot swarmplot pointplot clustermap tsplot palplot FacetGrid PairGrid JointGrid axes_style set_style plotting_context set_context set_color_codes reset_defaults reset_orig set
```
Comment (malmaud) [inline src/Seaborn.jl]: You don't need to change this line in this case, since you are manually defining those functions below.

**Review by malmaud — CHANGES_REQUESTED** [review]
Looks good! Just one little comment to address and I'll merge this.

File: src/Seaborn.jl (hunk 1 above)
Comment (josephmarturano) [reply src/Seaborn.jl]: Okay thanks, I removed `lineplot scatterplot relplot` from the delegate macro in line 66

**Review by malmaud — APPROVED** [review]

**Comment by malmaud** [comment]
Thanks!

**Pull request merged**